features/admin.branding.v1/components/custom-text/custom-text-fields.tsx

```diff
@@ -155,6 +163,94 @@ const CustomTextFields: FunctionComponent<CustomTextFieldsProps> = (props: Custo
         );
     };
 
+    const handleMultiAttributeLink = (): void => {
+        if (serverConfigurationConfig.dynamicConnectors === true) {
+            history.push(
+                AppConstants.getPaths().get(
+                    "GOVERNANCE_CONNECTOR_EDIT"
+                ).replace(
+                    ":categoryId",
+                    ServerConfigurationsConstants.
+                        ACCOUNT_MANAGEMENT_CONNECTOR_CATEGORY_ID
+                ).replace(
+                    ":connectorId",
+                    ServerConfigurationsConstants.
+                        MULTI_ATTRIBUTE_LOGIN_CONNECTOR_ID
+                )
+            );
+        } else {
+            history.push(AppConstants.getPaths().get("ALTERNATIVE_LOGIN_IDENTIFIER_EDIT"));
+        }
+    };
+
+    const getMultiAttributeLabel = (): string => {
+        if (serverConfigurationConfig.dynamicConnectors === true) {
+            return t("branding:connectors.multiAttributeLogin");
+        }
+        else {
+            return t("branding:connectors.alternativeLoginIdentifier");
+        }
+    };
+
+    const isMultiAttributeEnabled = (): boolean => {
+        if (serverConfigurationConfig.dynamicConnectors === true) {
+            return !serverConfigurationConfig.connectorsToHide
+                .includes(ServerConfigurationsConstants.MULTI_ATTRIBUTE_LOGIN_CONNECTOR_ID);
+        }
+        else {
+            return !serverConfigurationConfig.connectorsToHide
+                .includes(ServerConfigurationsConstants.ALTERNATIVE_LOGIN_IDENTIFIER);
+        }
+    };
+
+    const getHelperText = (hintKey: string, warningKey: string, fieldName: string): ReactElement => {
+        return i18n.exists(hintKey) || i18n.exists(warningKey)
+            ? (
+                <Hint>
+                    { i18n.exists(hintKey) && t(hintKey, { productName }) }
+                    { i18n.exists(warningKey) && (
+                        <>
+                            <br/><br/>
```

Comment:
Do we require this part? If this is used for spacing, let's use padding/margin instead